src/index.ts

```diff
@@ -271,14 +271,23 @@ async function appBoot() {
 			await migrateFromWpNowFolder();
 		}
 
+		const userData = await loadUserData();
+
+		if ( ! userData.sentryUserId ) {
+			userData.sentryUserId = crypto.randomUUID();
+			await saveUserData( userData );
+		}
+
+		console.log( 'Setting Sentry user ID:', userData.sentryUserId );
+		Sentry.setUser( { id: userData.sentryUserId } );
```

Comment:
Also, I would be more specific here too, e.g.
```suggestion
		Sentry.setUser( { studio-instance-id: userData.sentryUserId } );
```

It will be easier to understand in Sentry UI what the id is.

Reply:
`Sentry.setUser` doesn't accept arbitrary keys. The `id` parameter is standardized. See [docs](https://docs.sentry.io/platforms/javascript/guides/electron/enriching-events/identify-user/)

Reply:
Oh, in this case I definitely ok with naming 👍